docs/user/ai_model_serving.md

```diff
@@ -0,0 +1,517 @@
+will be configured with the `imagePullPolicy:` set to `Always` and the local image
+will be ignored. If you use any other tag, the `imagePullPolicy:` is set to `IfNotPresent`.
+
+```sh
+sudo podman build -t ovms-resnet50:test .
+```
+
+Example output:
+```
+STEP 1/3: FROM quay.io/microshift/busybox:1.37
+Trying to pull quay.io/microshift/busybox:1.37...
+Getting image source signatures
+Copying blob a46fbb00284b done   |
+Copying config 27a71e19c9 done   |
+Writing manifest to image destination
+STEP 2/3: RUN mkdir -p /models/1 && chmod -R 755 /models/1
+--> eacb7039436a
+STEP 3/3: RUN wget -q -P /models/1   https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin   https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml
+wget: note: TLS certificate validation not implemented
+COMMIT ovms-resnet50
+--> ac4606eb6cb3
+Successfully tagged localhost/ovms-resnet50:test
+ac4606eb6cb3e6be2fbee9d6bc271df212eb22e6a45a2c33394d9c73dc3bb4cf
+```
+
+Run the following command to make sure the image exists:
+```sh
+sudo podman images | grep ovms-resnet50
+```
+
+Example output:
+```
+localhost/ovms-resnet50          test          ac4606eb6cb3  3 minutes ago  27.5 MB
+```
+
+### Creating your namespace
+
+Use the following command to create a new namespace that will be used throughout this guide:
+```sh
+oc create ns ai-demo
+```
+
+### Deploying ServingRuntime to the workload's namespace
+
+First, you must select the ServingRuntime that supports the format of your model.
+Then, you need to create the ServingRuntime in your workload's namespace.
+
+If the cluster is already running, you can export the desired `ServingRuntime` to a file and tweak it.
+If they cluster is not running or you want to prepare a manifest, you can use
+original definition on the disk.
+
+For more information about ServingRuntimes refer to the
+[RHOAI documentation](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.17/html/serving_models/serving-large-models_serving-large-models#model-serving-runtimes_serving-large-models)
+
+#### Exporting ServingRuntime from a live cluster
+
+Use following command to list available ServingRuntimes:
+```sh
+oc get -n redhat-ods-applications servingruntimes
+```
+
+Run following command to save ServingRuntime definition to a YAML file:
+```sh
+oc get -n redhat-ods-applications servingruntimes kserve-ovms -o yaml > ovms.yaml
+```
+
+Edit the saved file and remove `.metadata.namespace` to allow creating the object in different namespace.
+Optionally, remove also other metadata:
+- `creationTimestamp`
+- `generation`
+- `resourceVersion`
+- `uid`
+
+Re-create the ServingRuntime in a custom namespace:
+```sh
+oc create -n ai-demo -f ./ovms.yaml
+```
+
+#### Create ServingRuntime based on installed manifests and release info
```

Comment:
I'm not sure we want this in our docs. Can we stay with the live-cluster-only part?